Presentation.Web/Presentation.Web.csproj

```diff
@@ -321,7 +324,11 @@
   <ItemGroup>
     <Compile Include="Controllers\API\KLEController.cs" />
     <Compile Include="Controllers\API\ReportController.cs" />
+    <Content Include="idp-metadata\kombit-saml_demo-brugervendtsystem.xml" />
```

Comment:
det er da ikke et navn vi kan leve med... vi er ikke demo systemet

Reply:
Se kommentarer til selve filen.. vi skal jo nok have begge metadatafiler og så er det config der peger den Id ud vi vil have skudt igang

Reply:
Ja, ovenstående er SAML config for KOMBIT test miljøet .. når vi laver deploy til prod skal der være en tilsvarende fil for KOMBI prod (og man skal kunne vælge hvilken i web.config/parameters)